.circleci/config.yml

```diff
@@ -125,6 +125,7 @@ jobs:
           paths:
             # In Next.js 8 it will be
             - ./node_modules/.cache/next-minifier
```

Comment:
Don't think it's useful to keep `.cache/next-minifier`. No?

Reply:
Yes, I will remove it and also squash it